apps/krystal-bull/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.7"
+
+services:
+  web:
+    image: bitcoinscala/oracle-server-ui:0.0.1-cc6ea29a-SNAPSHOT
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      ORACLE_SERVER_API_URL: "http://${APP_KRYSTAL_BULL_SERVER_IP}:9998/"
+      TOR_PROXY: socks5://${TOR_PROXY_IP}:${TOR_PROXY_PORT}
```

Comment:
What's this used for? Isn't this just a static web server running a browser app?

Reply:
This allows sending announcements and attestations to an Oracle Server through tor. You can toggle tor on/off and which Oracle Server you are using in the Configuration sidebar.

Reply:
Got it, thanks!